<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NetTest</class>
 <widget class="QWidget" name="NetTest">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>398</width>
    <height>113</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout">
   <item row="2" column="1">
    <widget class="QLineEdit" name="nettestStreamID">
     <property name="inputMask">
      <string/>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="maxLength">
      <number>5</number>
     </property>
     <property name="readOnly">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="nettestSeqNumber">
     <property name="inputMask">
      <string/>
     </property>
     <property name="text">
      <string>123</string>
     </property>
     <property name="readOnly">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="nettestTimeStamp">
     <property name="inputMask">
      <string/>
     </property>
     <property name="text">
      <string>456</string>
     </property>
     <property name="readOnly">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label_22">
     <property name="text">
      <string>Номер</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy">
      <cstring>nettestSeqNumber</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_24">
     <property name="text">
      <string>Временная метка</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy">
      <cstring>nettestTimeStamp</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="2" colspan="2">
    <widget class="QComboBox" name="cmbSeqNumberMode">
     <item>
      <property name="text">
       <string>Фиксированный</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Последовательный</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="1" column="2" colspan="2">
    <widget class="QComboBox" name="cmbTimeStampMode">
     <item>
      <property name="text">
       <string>Фиксированная</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Системный таймер</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="0" column="4">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_25">
     <property name="text">
      <string>ID потока</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy">
      <cstring>nettestStreamID</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <spacer>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>nettestSeqNumber</tabstop>
  <tabstop>nettestTimeStamp</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
